hack/lib/util/logs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+get_logging_pod_logs() {
+  local node
+  for node in $(oc get nodes --no-headers | awk '{print $1}') ; do
```

Comment:
the usual idiom for getting only the name of some object is to use `-o jsonpath='{.items[*].metadata.name}'` - so
```bash
for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}')
```

Reply:
thanks!